spec/controllers/enterprises_controller_spec.rb

```diff
@@ -112,6 +108,19 @@
       end
     end
 
+    it "resets order if the order cycle of the current order is no longer open or visible" do
+      order.distributor = distributor
+      order.order_cycle = order_cycle1
+      order.save
+      order_cycle1.update_attribute :orders_close_at, Time.zone.now
+
+      spree_get :shop, id: distributor
+
+      expect(controller.current_order.distributor).to eq(distributor)
+      expect(controller.current_order.order_cycle).to eq(order_cycle2)
+      expect(controller.current_order.line_items).to be_empty
+    end
```

Comment:
Do you think it's worth adding specs for the cases here? 

https://github.com/openfoodfoundation/openfoodnetwork/compare/master...Matt-Yorkley:enterprise-controller-stock

- Line item variants are all in stock
- Line item variants are not all in stock
- A line item's variant has been soft-deleted
- A line item's variant has been removed from the OC

Reply:
I was planning to do this as part of this PR but now that Pau moved it to Test Ready with the two approvals, we can probably get this into the release.
I will work on these specs anyway and create a new PR. I am a bit all over 10's of PRs, I may get to this only on Monday.

Reply:
IMO it's fine as is. The services seem mostly covered with unit tests and we might just need to increase the integration coverage at controller level a little bit.